lib/coverband/adapters/hash_redis_store.rb

```diff
@@ -88,9 +88,9 @@ def save_report(report)
 
       def coverage(local_type = nil)
         files_set = files_set(local_type)
-        @redis.pipelined {
-          files_set.map do |key|
-            @redis.hgetall(key)
+        @redis.pipelined { |pipeline|
+          files_set.each do |key|
```

Comment:
any reason the `map` moved to an `each`?

Reply:
Yes because the return value doesn't matter. `pipelined` doesn't care about what the block returns.

Reply:
gotch, yep all seems good.